src/main/java/bio/terra/service/snapshot/SnapshotTable.java

```diff
@@ -66,4 +67,25 @@ public SnapshotTable primaryKey(List<Column> primaryKey) {
     this.primaryKey = primaryKey;
     return this;
   }
+
+  @Override
+  public boolean equals(Object o) {
```

Comment:
It doesn't really make sense to have `equals()` and `hashcode()` on an object with mutable fields. Why was this change necessary?

Reply:
This is useful for testing.  This gives us a way to compare the full objects in tests